invenio_app_ils/ui/src/invenio_app_ils/common/components/Pagination/Pagination.js

```diff
@@ -0,0 +1,104 @@
+/*
+ * This file is part of React-SearchKit.
+ * Copyright (C) 2019 CERN.
+ *
+ * React-SearchKit is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Pagination as Paginator } from 'semantic-ui-react';
+import './Pagination.scss';
+
+export class Pagination extends Component {
+  constructor(props) {
+    super(props);
+    this.updatePage = this.props.updatePage;
+    this.renderElement = props.renderElement || this._renderElement;
+  }
+
+  onPageChange = activePage => {
+    this.updatePage(activePage);
+  };
+
+  _renderElement = (
+    currentPage,
+    currentSize,
+    totalResults,
+    onPageChange,
+    options
+  ) => {
+    const pages = Math.ceil(totalResults / currentSize);
+    const boundaryRangeCount = options.boundaryRangeCount;
+    const siblingRangeCount = options.siblingRangeCount;
+    const showEllipsis = options.showEllipsis;
+    const showFirst = options.showFirst;
+    const showLast = options.showLast;
+    const showPrev = options.showPrev;
+    const showNext = options.showNext;
+    const _onPageChange = (event, { activePage }) => {
+      onPageChange(activePage);
+    };
+
+    return (
+      <Paginator
+        activePage={currentPage}
+        totalPages={pages}
+        onPageChange={_onPageChange}
+        boundaryRange={boundaryRangeCount}
+        siblingRange={siblingRangeCount}
+        ellipsisItem={showEllipsis ? undefined : null}
+        firstItem={showFirst ? undefined : null}
+        lastItem={showLast ? undefined : null}
+        prevItem={showPrev ? undefined : null}
```

Comment:
Or you don't even need to destruct the options, pass them directly here, `options.showPrev`